scripts/load_aks_credentials.sh

```diff
@@ -0,0 +1,138 @@
+#!/bin/bash
```

Comment:
NITPICK: Could we re-name the scripts to maybe

1. load_kube_credentials.sh
2. save_kube_credentials.sh

as the code is  generic to the k8s clusters.